nautobot_dns_models/navigation.py

```diff
@@ -19,6 +19,6 @@
 menu_items = (
     NavMenuTab(
         name="Plugins",
```

Comment:
If this app requires Nautobot 2.2 or later as a minimum, can change this to `Apps`.